docs/immortal-empires/norsca.md

```diff
@@ -0,0 +1,30 @@
+# Norsca
+
+## Wulfrik the Wanderer - World Walkers
+
+* **Route I - Ragnarok**
+  * The End Times are nigh. Wulfrik has long cursed the gods for his predicament, but now realises he must pledge himself to one of them if he is to survive the coming apocalypse.
+  * Attain level 3 allegiance with a Chaos God.
+   * Defeat all of the Challengers sent against you.
+  * **Reward**:
+	* _Seafang Empowered_: Campaign movement range +30% (factionwide), enables teleportation for Wulfrik from port cities he owns (possible?)
+_Reward flavour text:_ Seafang's figurehead quickly absorbed the blood of the slain challengers that were smeared upon it. The ship now glows with eldritch power like never before.
+
+* **Route II - The World Walker**
+  * After a drunken boast made in sight of the Dark Gods, Wulfrik was cursed to forever wander the earth and seek out the fiercest challengers and most monstrous creatures and slay them in single combat.
+  * Occupy at least 5 of the following settlements:
+	* Couronne, Miragliano, Karaz-a-Karak, Altdorf, Castle Drakenhof, Black Crag, Kislev, Naggarond, Lothern, Hexoatl, Skavenblight, Khemri, The Awakening
+  * **Reward**:
+	* _The Eternal Challenger_: Enemy leadership -%10, enemy units near Wulfrik periodically Rampage (attacks the closest enemy and ignores any orders given).
+_Reward flavour text:_ With his Gift of Tongues Wulfrik curses foes and goads them into recklessness. Whether he's besmirching a Dwarf's honour, questioning a Bretonnian's lineage or humiliating a haughty Elf - none can ignore his insults for long.
+
+* **Route III - Where's My Palico?**
+  * Wulfrik leads his World Walkers across the world in search of mighty beasts to slay. Though the rewards are great, so too are the risks.
+  * Complete 6 Monster Hunts
+  * The Norse adventurer Losteriksson was the first Norscan to land on the coast of Lustria and live to tell the tale. The settlement he founded still stands to this day, though it is under dire threat from the Lizardmen of the jungle.

```

Comment:
I sincerely do not like mixing Monster Hunts and Lustria stuff. They feel like two completely independent routes merged into one.

Reply:
Route 1 and route 3 objectives are going to be identical to Throgg's. This was my way of differentiating them.

If we don't care about making them unique then we can delete the Skeggi part.

Reply:
Problem is Throgg is a troll and doesn't make a lot of sense for him to want to go to Skeggi. Best to just delete it entirely then and maybe another Norsca lord will come out one day that fits.

Reply:
Yeah, but it doesn't make any sense for Throgg to lead an army of human marauders anyway, so we play with what we have. We cna leave it for future lords tho, as I really expect a DLC here sooner than later.

Reply:
My initial thinking was they both got monster hunts themed to them but still optional to player. Like Wulfrik goes to Skeggi, while he's there, he's right near the Lustria monster hunt. Throgg goes to Greenskin lands to gather more trolls for his troll army, while he's there, he gets the Badlands monster hunt, etc.